common/locales/en/settings.json

```diff
@@ -108,10 +108,12 @@
     "emailNotifications": "Email Notifications",
     "wonChallenge": "You won a Challenge!",
     "newPM": "Received Private Message",
+    "newPMInfo": "New Message from <%= name %>: <%= message %>",
     "sentGems": "Sent gems!",
     "giftedGems": "Gifted Gems",
-    "giftedGemsInfo": "<%= amount %> Gems - by <%= name %>",
+    "giftedGemsInfo": "<%= name %> gifted you <%= amount %> Gems",
     "giftedSubscription": "Gifted Subscription",
+    "giftedSubscriptionInfo": "<%= name %> gifted you a <%= months %> Subscription",
```

Comment:
Some of these changes seem unrelated to the rest of the PR.  Specifically, the changes regarding the `{newPM,giftedGems,giftedSubscription}Info` strings.